redis-utils-lib/test/cmr/redis_utils/test/test_redis_cache.clj

```diff
@@ -38,3 +40,96 @@
       (wcar* (carmine/set (str "get-keys-pattern-2#-" x) x)))
     (is (= (config/redis-max-scan-keys)
            (count (redis/get-keys "get-keys-pattern-2#-*"))))))
+
+(def field-value-map
+  {"C1200000001-PROV1"
+   {:concept-id "C1200000001-PROV1",
+    :revision-id 4,
+    :native-format {:format :umm-json, :version "1.17.3"},
+    :echo10 "<Collection><ShortName>Mapping...",
+    :dif "<DIF xmlns=\"http://gcmd.gsfc.n...",
+    :dif10 "<DIF xmlns=\"http://gcmd.gsfc.n...",
+    :iso19115 "<gmi:MI_Metadata xmlns:xs=\"htt...",
+    :umm-json_1.17.3 "{      \"DataLanguage\" : \"Engli..."}
+   "C1200000002-PROV1"
+   {:concept-id "C1200000002-PROV1",
+    :revision-id 4,
+    :native-format {:format :umm-json, :version "1.17.3"},
+    :echo10 "<Collection><ShortName>SWOT_L2...",
+    :dif "<DIF xmlns=\"http://gcmd.gsfc.n...",
+    :dif10 "<DIF xmlns=\"http://gcmd.gsfc.n...",
+    :iso19115 "<gmi:MI_Metadata xmlns:xs=\"htt...",
+    :umm-json_1.17.3 "{  \"AdditionalAttributes\" : [ ..."}})
+
+(def single-field-value-map
+  {"C1200000003-PROV1" {:concept-id "C1200000003-PROV1",
+                        :revision-id 4,
+                        :native-format {:format :umm-json, :version "1.17.3"},
+                        :echo10 "<Collection><ShortName>SWOT_L2...",
+                        :dif "<DIF xmlns=\"http://gcmd.gsfc.n...",
+                        :dif10 "<DIF xmlns=\"http://gcmd.gsfc.n...",
+                        :iso19115 "<gmi:MI_Metadata xmlns:xs=\"htt...",
+                        :umm-json_1.17.3 "{  \"AdditionalAttributes\" : [ ..."}})
+
+(deftest redis-hash-cash-test
+  (let [cache-key :test-hash-cache
+        rhcache (redis-hash-cache/create-redis-hash-cache {:keys-to-track [cache-key]})]
```

Comment:
so you could create multiple caches in one use? I guess if your code needed to track two different things?